packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -21,17 +25,41 @@ function getBaseMafsGraphProps(): MafsGraphProps {
         box: [400, 400],
         step: [1, 1],
         gridStep: [1, 1],
-        snapStep: [1, 1],
         markings: "graph",
+        containerSizeClass: "small",
+        showTooltips: false,
+        labels: ["x", "y"],
+        dispatch: () => {},
+        state: {
+            type: "segment",
+            hasBeenInteractedWith: false,
+            coords: [],
+            markings: "none",
+            snapStep: [1, 1],
+            range: [
+                [-10, 10],
+                [-10, 10],
+            ],
+        },
+    };
+}
+
+function getBaseStatefulMafsGraphProps(): StatefulMafsGraphProps {
```

Comment:
Questions: Is this separate function needed just to more closely match the types required by certain tests? And this stateful one is the same as the old base one was, right?

Reply:
Yep!